src/App.js

```diff
@@ -1,30 +1,38 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
-import { Switch, Route } from 'react-router-dom';
+import { Link, Switch, Route, Redirect } from 'react-router-dom';
 import FirebaseTesting from './pages/FirebaseTest';
 import FooterTabs from './components/FooterTabs';
 import ShoppingList from './pages/ShoppingList';
 import AddItem from './pages/AddItem';
+import NewListButton from './components/NewListButton';
+import * as ls from 'local-storage';
 
 function App() {
+	const [token, setToken] = useState(ls.get('shoppingListToken'));
+	console.log({ token });
```

Comment:
Let's try to keep console.logs out of production if possible. This is something most professional dev teams will push for as well. You can totally pop into the storage tab in your console to get these details instead! :-)